src/services/pcn-nat/src/Nat_dp_common.c

```diff
@@ -52,9 +52,9 @@ struct st_k {
   uint16_t src_port;
   uint16_t dst_port;
   uint8_t proto;
-};
+} __attribute__((packed));
```

Comment:
Matteo, can you please add a brief comment in the code stating why this attribute is needed?
E.g.  /* attribute packet forces alignment for this structure; otherwise segmentation fault may happen when the userspace component tries to get access to that above data */

Reply:
Sure, comments added.